pkgs/tools/backup/gphotos-sync/default.nix

```diff
@@ -30,36 +16,36 @@ py.pkgs.buildPythonApplication rec {
     owner = "gilesknap";
     repo = "gphotos-sync";
     rev = version;
-    sha256 = "0mnlnqmlh3n1b6fjwpx2byl1z41vgghjb95598kz5gvdi95iirrs";
+    hash = "sha256-lLw450Rk7tIENFTZWHoinkhv3VtctDv18NKxhox+NgI=";
   };
 
   patches = [
     ./skip-network-tests.patch
   ];
 
-  propagatedBuildInputs = with py.pkgs; [
+  postPatch = ''
+    sed -i 's/psutil==5.9.4/psutil/' setup.cfg
+    sed -i 's/exif==1.6.0/exif/' setup.cfg
+    '';
```

Comment:
You may use `pythonRelaxDepsHook`

Reply:
Thanks, didn't knew about this :)

Fixed.